### PR TITLE
Set up single-page App using react routing

### PR DESCRIPTION
**Pull request by karelvanoordt**
- Set up React routing
- Added styling using inline css and modules.
- Update "Quotes" page using random quote API.

### CODE REVIEW

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: INVALID ❌ ❌ 
Hi @karelvanoordt , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project with linter errors. Please ensure all linters are passing before you request for a review.
Screenshot 2021-12-01 at 15 03 00

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Davidosky007** [comment]
## Hi @karelvanoordt 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has failing linter checks kindly fix the linter errors and submit them for review 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/App.js [hunk 1]
```diff
@@ -1,20 +1,26 @@
 import React from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+  Routes,
+} from 'react-router-dom';
 import './index.css';
-import Calculator from './components/calculator';
+import Navbar from './components/Navbar';
+import Calculator from './components/Calculator';
```
Comment (DeliceLydia) [inline src/App.js:9]: ## Optional
- [ ] Please change this import the file is `calculator` not `Calculator` it was causing the server not to run

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉 

## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Kudos on using internal API for quotes🎉🎉
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (DeliceLydia) [reply src/App.js:9]: - [ ] Remember to add the live version in the README

**Pull request merged**